app/models/vote.rb

```diff
@@ -0,0 +1,4 @@
+class Vote < ApplicationRecord
+  belongs_to :user
+  belongs_to :work
```

Comment:
You need a validation to keep people from double-voting

```suggestion
  belongs_to :work
  
  validates :user_id, uniqueness: { scope: :work_id }
```